api/api.go

```diff
@@ -99,6 +99,7 @@ type auth struct {
 func (a *auth) authenticate(handle func(http.ResponseWriter, *http.Request)) http.HandlerFunc {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
+
 		if r.Header.Get(internalToken) == a.key {
 			ctx = context.WithValue(ctx, internalToken, true)
```

Comment:
golang should not use basic type untyped string as key in context.withvalue

See: https://medium.com/@matryer/context-keys-in-go-5312346a868d

Simply you can do the following:

1) Create ca contextKey variable
```
type contextKey string

func (c contextKey) String() string {
	return string(c)
}
```
2) Replace line 104 with `ctx = context.WithValue(ctx, contextKey(internalToken), true)`